AWS_RDS_IAM_SUPPORT.md

```diff
@@ -0,0 +1,206 @@
+# AWS RDS IAM Authentication Support for PostgreSQL Scaler
```

Comment:
not sure where should we put this @JorTurFer 🤔 ? Should we put in the keda-docs?